data/codelists/codelist-59.yml

```diff
@@ -6,7 +6,16 @@
   '03': ReducedPriceApplicableWhenTheItemIsPurchasedAsPartOfASetOrSeriesOrCollection
   '04': VoucherPrice
   '05': ConsumerPrice
-  '06': CorporatePrice
+  '06': CorporatePrice # Actually CorporateLibraryEducationPrice but keeping CorporatePrice for backwards compatibility
   '07': ReservationOrderPrice
   '08': PromotionalOfferPrice
   '09': LinkedPrice
+  '10': LibraryPrice
+  '11': EducationPrice
+  '12': CorporatePrice
```

Comment:
Now we have two `CorporatePrice` because `06` changed to `CorporateLibraryEducationPrice` but we keep it to `CorporatePrice` for backwards compatibility. Or shouldn't we?

Any opinions here? Because `CorporateLibraryEducationPrice` looks like something that should not be allowed on retail products just as much as `LibraryPrice` (and maybe others).